README.md

```diff
@@ -181,26 +181,27 @@ You can:
-| LOG_TYPE              | Output log type, supports pretty or json.                                     | pretty                    |
+| Variable Name          | Description                                                                   | Default                   |
+|------------------------|-------------------------------------------------------------------------------|---------------------------|
+| DB_HOST                | Host name of the MySQL server                                                 | gql-schema-registry-db    |
+| DB_USERNAME            | Username to connect to MySQL                                                  | root                      |
+| DB_SECRET              | Password used to connect to MySQL                                             | root                      |
+| DB_PORT                | Port used when connecting to MySQL                                            | 3306                      |
+| DB_NAME                | Name of the MySQL database to connect to                                      | schema-registry           |
+| DB_EXECUTE_MIGRATIONS  | Controls whether DB migrations are executed upon registry startup or not      | true                      |
+| REDIS_HOST             | Host name of the Redis server                                                 | gql-schema-registry-redis |
+| REDIS_PORT             | Port used when connecting to Redis                                            | 6379                      |
+| REDIS_SECRET           | Password used to connect to Redis                                             | Empty                     |
+| ASSETS_URL             | Controls the url that web assets are served from                              | localhost:6001            |
+| NODE_ENV               | Specifies the environment. Use _production_ to load js/css from `dist/assets` | Empty                     |
+| ASYNC_SCHEMA_UPDATES   | Specifies if async achema updates is enabled                                  | false                     |
+| KAFKA_BROKER_HOST      | Host name of the Kafka broker, used if ASYNC_SCHEMA_UPDATES = true            | gql-schema-registry-kafka |
+| KAFKA_BROKER_PORT      | Port used when connecting to Kafka, used if ASYNC_SCHEMA_UPDATES = true       | 9092                      |
+| KAFKA_SCHEMA_TOPIC     | Topic with new schema                                                         | graphql-schema-updates    |
+| KAFKA_QUERIES_TOPIC    | Topic with new schema                                                         | graphql-queries           |
+| LOG_LEVEL              | Minimum level of logs to output                                               | info                      |
+| LOG_TYPE               | Output log type, supports pretty or json.                                     | pretty                    |
+| LOG_STREAMING_ENABLED  | Controls whether logs are streamed over Redis to be presented in UI           | false                     |
```

Comment:
I'd suggest to enable it by default to have richest experience **by default**

Reply:
Or what do you think.. should we move to most minimal version by default (no redis, no kafka.. maybe even no mysql -> sqlite?)

Reply:
> I'd suggest to enable it by default to have richest experience **by default**

Sounds reasonable to me. As long as there's a configuration to disable it, I don't mind having to disable it within our deployments.